### ISSUE TITLE
Could not pull latest image

### ISSUE THREAD

**Issue by izhangzhihao** [bitnami/bitnami-docker-airflow-worker#11]
```
> docker pull docker.io/bitnami/airflow:2.0.2-debian-10-r0
Error response from daemon: manifest for bitnami/airflow:2.0.2-debian-10-r0 not found: manifest unknown: manifest unknown
```

```
> docker pull docker.io/bitnami/airflow:2.0.1-debian-10-r66
Error response from daemon: manifest for bitnami/airflow:2.0.1-debian-10-r66 not found: manifest unknown: manifest unknown
```

```
> docker pull docker.io/bitnami/airflow:2.0.1-debian-10-r65
Error response from daemon: manifest for bitnami/airflow:2.0.1-debian-10-r65 not found: manifest unknown: manifest unknown
```

```
> docker pull docker.io/bitnami/airflow:2.0.1-debian-10-r64

Error response from daemon: manifest for bitnami/airflow:2.0.1-debian-10-r64 not found: manifest unknown: manifest unknown
```

```
docker pull docker.io/bitnami/airflow:2.0.1-debian-10-r63
2.0.1-debian-10-r63: Pulling from bitnami/airflow
f87be781ad7c: Already exists
7ae7010f0bd9: Pulling fs layer
3c3ec5580aeb: Pulling fs layer
f942f66b0398: Pulling fs layer
140d50acb539: Waiting
8c1c0b010396: Waiting
fa15b1c64c8f: Waiting
f087779da9fa: Waiting
d85610059623: Waiting
fc838a4be750: Waiting
1b72e80a90ed: Waiting
e588e6b61e60: Waiting
380bc3bdcd56: Waiting
5ef694f3244a: Waiting
92712900a25e: Waiting
65baaae67156: Waiting
```

**Comment by izhangzhihao** [comment]
The version `2.0.1-debian-10-r64` release 4 days ago.

**Comment by carrodher** [comment]
Hi, there are 3 different airflow images:

**1. `bitnami/airflow`**
- At this moment, the latest tag in branch 2.X is `2.0.1-debian-10-r63` (see [dockerhub](https://hub.docker.com/r/bitnami/airflow/tags?page=1&ordering=last_updated) or [github](https://github.com/bitnami/bitnami-docker-airflow/releases))

**2. `bitnami/airflow-worker`**
- At this moment, the latest tag in branch 2.X is `2.0.2-debian-10-r0` (see [dockerhub](https://hub.docker.com/r/bitnami/airflow-worker/tags?page=1&ordering=last_updated) or [github](https://github.com/bitnami/bitnami-docker-airflow-worker/releases))

**3. `bitnami/airflow-scheduler`**
- At this moment, the latest tag in branch 2.X is `2.0.2-debian-10-r0` (see [dockerhub](https://hub.docker.com/r/bitnami/airflow-scheduler/tags?page=1&ordering=last_updated) or [github](https://github.com/bitnami/bitnami-docker-airflow-scheduler/releases))

For `bitnami/airflow` higher revisions than `2.0.1-debian-10-r63` (like `-r64`, `-r65` or `-r66`) are not released (nor are going to be released). The following tag for `bitnami/airflow` is going to be `2.0.2-debian-10-r0` this version is not released because it is failing during the test process we have before release new images, where the containers and Helm Charts are tested.

Once fixed the issue in the test & release pipeline the new `bitnami/airflow` container image will be released and the Helm Chart will be updated accordingly. 

In the end, the confusion appeared because you are comparing tags from different images since `-r64`, `-r65` or `-r66` are valid tags for `bitnami/airflow-worker` but not for `bitnami/airflow`. As they are different images the revisions can be different as well.

**Comment by izhangzhihao** [comment]
Any update on that?

**Comment by carrodher** [comment]
Not yet, we are still working on some changes in the Helm Chart.
The Airflow Helm Chart uses Redis as a subchart and we just released a new major version of the Redis Helm Chart (see https://github.com/bitnami/charts/pull/6102), so we need to do some manual changes in order to adapt Airflow to the new Redis major.

Usually, the process to release new versions is automatic, but as we are testing the Helm Charts before publishing the container images some times there are other unrelated changes that need to be addressed. Probably the issue is going to be solved during the next week.

**Comment by izhangzhihao** [comment]
Maybe we could still use the current Redis version until the Redis Helm Chart fixed? So I could process my work based on the latest Airflow feature.

**Comment by izhangzhihao** [comment]
I mean, we could at least make the `bitnami/airflow:2.0.2-debian-10-r0` available, so I could use the release.

**Comment by carrodher** [comment]
I am sorry but our internal process tests all the images used in the Helm Charts before publishing them.
In this case, the `bitnami/airflow` image is used in the Bitnami Airflow Helm Chart, so although the issue is not directly related to the image new version but the Helm Chart itself, the image release is going to be blocked by the automated test & release pipeline.
We need to ensure everything works as expected before release a new image and that means to check that the Helm Chart works with this image. At this moment is something we can't guarantee since the Helm Chart is failing (apparently due to another issue we are working on), but we can't bypass this process.

**Comment by izhangzhihao** [comment]
```
docker pull docker.io/bitnami/airflow:2.0.2-debian-10-r0
2.0.2-debian-10-r0: Pulling from bitnami/airflow
3cc1ae8ac7e0: Pulling fs layer
a65d37d57ce4: Pulling fs layer
b2699768246a: Pulling fs layer
c47f4f03984d: Waiting
c1baf99031aa: Waiting
68d1f09b4ee2: Waiting
ef86359dbe1a: Waiting
886006c88a2f: Waiting
fbef30766e26: Waiting
dcf637fabbd4: Waiting
eafff22e6690: Waiting
18e20a3d5504: Waiting
8f53055ddba9: Waiting
165b8ab37605: Waiting
e94c0d6593c3: Waiting
84a8ab63ee2e: Waiting
```
